backend/routers/userRouter.js

```diff
@@ -7,4 +8,17 @@ router.post('/registerUser', userController.registerUser)
 router.post('/loginUser', userController.loginUser)
 router.get('/getUser', auth, userController.getUser)
 
+router.get('/validate-token', (req, res) => {

```

Comment:
Create a new api on userRouter called /validate-token that accepts the authorization token as parameter to verify jwt token